<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="1" hostname="DESKTOP-65LSBDS" failures="1" tests="5" name="Personal Loan" time="0.091" errors="0" timestamp="22 Apr 2020 20:09:52 GMT">
  <testcase classname="TestNGwithRahul.Day1" name="PersonalLoanSIGNIN" time="0.011"/>
  <testcase classname="TestNGwithRahul.Day1" name="PersonalLoanSIGNIN" time="0.012"/>
  <testcase classname="TestNGwithRahul.Day1" name="PersonalLoanSIGNIN" time="0.002"/>
  <testcase classname="TestNGwithRahul.Day1" name="Test2" time="0.002"/>
  <testcase classname="TestNGwithRahul.Day2" name="PersonalLoan" time="0.007">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at TestNGwithRahul.Day2.PersonalLoan(Day2.java:15)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:835)
... Removed 20 stack frames]]>
    </failure>
  </testcase> <!-- PersonalLoan -->
  <testcase classname="TestNGwithRahul.Day1" name="Test1" time="0.0">
    <ignored/>
  </testcase> <!-- Test1 -->
</testsuite> <!-- Personal Loan -->
